Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+//https://github.com/feedhenry/fh-pipeline-library
+
+fhBuildNode {
+    stage('Install Dependencies') {
+        dir('fh-messaging') {
+            npmInstall {}
+        }
+        dir('fh-metrics') {
+            npmInstall {}
+        }
+    }
+
+    stage('Build') {
+        dir('fh-messaging') {
+            gruntCmd {
```

Comment:
gruntBuild is running the archive step but I think that can only be executed once in any pipeline https://github.com/feedhenry/fh-pipeline-library/blob/master/vars/gruntBuild.groovy#L16.

For consistency I guess we should use "--only-bundle-deps", will update.